jans-cedarling/cedarling/src/init/policy_store.rs

```diff
@@ -61,5 +63,27 @@ pub(crate) fn load_policy_store(
         },
     };
 
+    // Check if there are any trusted issuers in the policy
+    if let Some(trusted_issuers) = &policy.trusted_issuers {
+        for issuer in trusted_issuers {
+            if let Some(metadata) = &issuer.token_metadata {
+                let tokens_with_role_mapping: Vec<TokenKind> = metadata
+                    .iter()
+                    .filter_map(|x| x.role_mapping.as_ref().map(|_| x.kind.clone()))
+                    .collect();
+
+                // If there are more than one token with `role_mapping`, return an error
+                if tokens_with_role_mapping.len() > 1 {
+                    let token_kinds: String = tokens_with_role_mapping
+                        .iter()
+                        .map(|token| token.to_string())
+                        .collect::<Vec<_>>()
+                        .join(", ");
+                    return Err(LoadPolicyStoreError::MultipleRoleMappings(token_kinds));
```

Comment:
Using MultipleRoleMappings(Vec<String>)


```suggestion
                    let token_kinds = tokens_with_role_mapping
                        .iter()
                        .map(|token| token.to_string())
                        .collect::<Vec<String>>();
                    return Err(LoadPolicyStoreError::MultipleRoleMappings(token_kinds));
```

Reply:
resolved in [782bcee](https://github.com/JanssenProject/jans/pull/9893/commits/782bceebf6159f73e8558f9ec5ee48638bb4b1c1)